aptos-move/framework/aptos-token-objects/sources/token.move

```diff
@@ -989,17 +1046,43 @@ module aptos_token_objects::token {
         assert!(vector::length(&event::emitted_events<collection::Burn>()) == 1, 0);
     }
 
+    #[test(creator = @0x123)]
+    fun test_change_collection_name(creator: &signer) {
+        let collection_name = string::utf8(b"collection name");
+        let token_name = string::utf8(b"token name");
+
+        let constructor_ref = &create_fixed_collection(creator, collection_name, 5);
+        let collection = get_collection_from_ref(&object::generate_extend_ref(constructor_ref));
+        let mutator_ref = collection::generate_mutator_ref(constructor_ref);
+
+        // Create initial token with current collection name
+        create_token_with_collection_helper(creator, collection, token_name);
+
+        // Set new collection name
```

Comment:
sorta minor nit, these comments are not helpful, because if you can't understand `collection::set_name` you will have difficulty with the comment...

Reply:
Let me remove these comments which don't add context.